diff --git a/dist/connect.js b/dist/connect.js
index 0f98a0b6dc46df646288cc3e2d2cb61edc4a2f44..9aa4131b9a86068de1f8c2b8f055398ce4c49a0b 100644
--- a/dist/connect.js
+++ b/dist/connect.js
@@ -43,7 +43,9 @@ class Connect extends irc_framework_1.Client {
             host: this.host,
             port: this.port,
             nick: this.nickname,
-            username: params.nickname || 'xdccJS',
+            username: "xdccMule",
+            gecos: "xdccMule",
+            version: "mIRC v6.35 Khaled Mardam-Bey",
             auto_reconnect_max_wait: 0,
             auto_reconnect_max_retries: 0,
             ssl: this.tls.enable,
@@ -136,7 +138,7 @@ class Connect extends irc_framework_1.Client {
     }
     turnOff(
     // eslint-disable-next-line no-unused-vars
-    rawListener, 
+    rawListener,
     // eslint-disable-next-line no-unused-vars
     noticeListener, timeout) {
         this.off('raw', rawListener);
